Directory.Build.props

```diff
@@ -1,3 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
+  <PropertyGroup>
+    <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  <PropertyGroup>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      https://ci.appveyor.com/nuget/benchmarkdotnet;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json;
```

Comment:
what packages are being restored from dotnet.myget.org and dotnet-tools-internal?  Are those sources needed?

Reply:
@adamsitnik would know better for the dotnet.myget.org, I just copied that from the nuget.config in the micro benchmark directory. dotnet-tools-internal is probably unnecessary. Removing.